src/main/scala/org/broadinstitute/dsde/firecloud/HealthChecks.scala

```diff
@@ -28,28 +28,44 @@ class HealthChecks(app: Application, registerSAs: Boolean = true)
       AccessToken(app.googleServicesDAO.getAdminUserAccessToken))
 
   private def maybeRegisterServiceAccount(name: String, token: AccessToken): Future[Option[String]] = {
-    val lookup = manageRegistration(name, app.samDAO.getRegistrationStatus(implicitly(token)))
+    val lookup = manageRegistration(name, app.samDAO.getRegistrationStatus(token))
     lookup flatMap {
-      case Some(_) if registerSAs =>
+      case Some(err) if registerSAs =>
+        logger.warn(s"SA registration lookup found: $err")
         logger.info(s"attempting to register $name ...")
-        manageRegistration(name, app.samDAO.registerUser(implicitly(token)))
+        manageRegistration(name, app.samDAO.registerUser(token))
```

Comment:
`implicitly`  is not needed here - the sam dao code may have changed since this was written, or we didn't need `implicitly` in the first place, I don't know